test/passing/break_cases.ml

```diff
@@ -0,0 +1,25 @@
+let f x = function
```

Comment:
can you add examples with or_pattern ?

Reply:
I think it would be good to also add some tests to cover the case where there is a nested or-pattern that is wide enough that it breaks even without `--sparse`. And to add a couple tests that include `when` guards in both modes as there is some interaction with breaking.

Reply:
Thoses cases are now covered in the break_cases(_sparse) files.